@xen-orchestra/web-core/lib/components/progress-bar/VtsProgressBar.vue

```diff
@@ -40,7 +41,11 @@ const progress = useProgress(
 
 const { percentageCap, percentage, fillWidth } = progress
 
-const legend = useProgressToLegend(() => legendType, label, progress)
+const legend = useProgressToLegend(
+  () => legendType,
+  () => label,
```

Comment:
```suggestion
  () => label ?? '',
```
if you decide to make the label optional